### PR TITLE
Adding getir.com

### PR DESCRIPTION
**Pull request by napolux**
Adding getir as remote company, at least for software engineering roles.

### CODE REVIEW

**Review by edoardocostantinidev — APPROVED** [review]

File: data/getir.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "name": "Getir",
+  "url": "https://getir.com/it/",
+  "career_page_url": "https://apply.workable.com/getir/",
+  "type": "Product",
+  "categories": ["food_delivery"],
```
Comment (alessandromr) [inline data/getir.json:6]:
```suggestion
  "categories": ["B2C"],
```

**Review by alessandromr — CHANGES_REQUESTED** [review]

File: data/getir.json (hunk 1 above)
Comment (napolux) [reply data/getir.json:6]: 👍

File: data/getir.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "name": "Getir",
+  "url": "https://getir.com/it/",
+  "career_page_url": "https://apply.workable.com/getir/",
+  "type": "Product",
+  "categories": ["B2C"],
```
Comment (alessandromr) [inline data/getir.json:6]:
```suggestion
  "categories": ["cloud_software"],
```

Comment (alessandromr) [reply data/getir.json:6]: Sorry, I suggested the wrong code. @napolux

Comment (napolux) [reply data/getir.json:6]: 👯‍♂️

Comment (napolux) [reply data/getir.json:6]: So what's the final version of the file I have to commit? I'm a bit confused now :D

**Review by alessandromr — APPROVED** [review]

**Pull request merged**